app/ts/components/TransferPage/Validation.tsx

```diff
@@ -27,7 +27,12 @@ export const TransferValidation = (props: Props) => {
 					<div>
 						<strong>Warning:</strong> Recipient is a Token address
 					</div>
-					<div class='leading-tight text-white/75 text-sm'>The recipient address provided is a known token address and may be unintentional. Tokens sent to this address may be unrecoverable.</div>
+					<div class='leading-tight text-white/75 text-sm mb-4'>The recipient address provided is a token contract address and may result in a loss of funds.</div>
+
+					<label class='flex gap-2 items-center'>
+						<input type='checkbox' required />
+						<span>I understand that this might result in loss of funds. Proceed anyway.</span>
```

Comment:
```suggestion
						<span>I understand that this will probably result in the loss of {props.data.value.amount} {props.data.value.symbol}.</span>
```